core/frontend/vue.config.js

```diff
@@ -49,6 +49,9 @@ module.exports = {
       '^/network-test': {
         target: SERVER_ADDRESS,
       },
+      '^/ping': {
```

Comment:
but the commits convention that we are following, the commit body should contains `frontend:`.